README.md

```diff
@@ -500,6 +500,7 @@ CMS are a practical way to setup a website with many features. CMS often come wi
 - [Sphido](http://www.sphido.org/) - Fast, lightweight, flat file CMS for PHP. ([Source Code](https://github.com/sphido/cms)) `MIT` `PHP`
 - [SPIP](http://www.spip.net/fr) - Publication system for the Internet aimed at collaborative work, multilingual environments, and simplicity of use for web authors. ([Source Code](https://core.spip.net/projects/spip/repository)) `GPL-2.0` `PHP`
 - [Subrion](http://www.subrion.org) - Subrion is a free open source content management system that allows you to build websites for any purpose. Yes, from blog to corporate mega portal. ([Demo](http://demos.subrion.com), [Source Code](https://github.com/intelliants/subrion)) `GPL-3.0` `PHP`
+- [Tent](http://secfirst.org/) - Mobile CMS using Markdown files on GitHub as the authoritative data source. [Source Code](https://github.com/securityfirst/tent)) `GPL-3.0` `Go`
```

Comment:
Please use the Github repository link as main link, http://secfirst.org/ has no information relevant to Tent.